Source/Models/SwishError.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import Argo
+
+public enum SwishError {
+  case SessionError(NSError)
+  case ServerError(code: Int, json: AnyObject)
+  case JSONSerializationError(NSError)
+  case ArgoError(DecodeError)
```

Comment:
Yeah, for sure. Would you just chop off `Error` at the end of the specific cases?